scripts/upload-apk.sh

```diff
@@ -20,7 +20,7 @@ for file in android*; do
     if [[ $file =~ ".aab" ]]; then
         mv $file badge-magic-$TRAVIS_BRANCH-$file
     else
-        mv $file badge-magic-$TRAVIS_BRANCH-${file:4}
+        mv $file badge-magic-$TRAVIS_BRANCH-${file:8}
```

Comment:
Hmm I thought you would understand how brittle this is

Reply:
I just noticed that the names are not correct in the apk branch when you posted a link in #660 xD

Reply:
Yes, it is because this method of renaming is error prone

Reply:
Let me find the first index of - and substring. I'll update it in sometime